python/tvm/relay/frontend/pytorch.py

```diff
@@ -3713,6 +3713,7 @@ def from_pytorch(
     custom_convert_map=None,
     default_dtype="float32",
     use_parser_friendly_name=False,
+    return_int8_weight=False,
```

Comment:
How about `keep_quantized_weight`? IIUC, this is effective when the PyTorch model is already quantized. `return_int8_weight` might confuse users and let them feel this flag can do quantization for them.

Reply:
Thanks, I think this is a good suggestion.